Document/0x06b-Basic-Security-Testing.md

```diff
@@ -278,6 +278,18 @@ The tool has the following global options (list them via the `show options` comm
 - **SAVE_HISTORY**: If set to "true," the command history will persist across sessions.
 - **VERBOSE, DEBUG**: If set to "true," this will enable verbose and debug logging, respectively.
 
+**Trouble shooting**
+
+Please pay attention, in order to use the modules in Needle. You might have to install its dependencies by using this command in Neelde.
```

Comment:
`Neelde` should be `Needle`.

I would remove "Please pay attention", as it doesn't add much.

I think that the following parts are particularly related: "in order to use the modules in Needle" and "You might have to install its dependencies". I would put them in the same sentence.

```suggestion
In order to use the modules in Needle, you may have to install its dependencies. Use this command in Needle:
```